app/Http/Controllers/Auth/AuthController.php

```diff
@@ -22,10 +23,10 @@ public function __construct()
      *
      * @return \Illuminate\Http\JsonResponse
      */
-    public function login()
+    public function login(LoginRequest $request)
     {
         $credentials = request(['email', 'password']);
```

Comment:
request->validated()

Reply:
Kada se napravi custom request mislim da on sam u pozadini radi validaciju. Kada posaljem nevalidne podatke dobijam 422 entity not processible

Reply:
radi, i to je okej, ali praksa je neka da se koristi ovaj `validated()` da ti izvuce samo izvalidirane podatke ( znaci ne radi ovo validaciju, nego vraca te validne podatke), ne bih ti znao reci zasto, nisam pretjerano zalazio u to, mozes iscackati u dokumentaciji ako te zanima sta kako, nisam imao vremena da se cimam oko toga